contracts/YearnCrvTwoTokens.sol

```diff
@@ -0,0 +1,122 @@
+pragma solidity ^0.8.5;
```

Comment:
Any reason not to bump this to `0.8.6` if we're using that version in Hardhat Config?

Reply:
Ah yep. I only bumped it in this one contract, since others have already been deployed at earlier versions. Done in a141f49